sunpy/conftest.py

```diff
@@ -130,7 +130,19 @@ def pytest_runtest_setup(item):
             pytest.skip("skipping remotedata tests as pytest-remotedata is not installed")
 
 
+def pytest_configure(config):
+    """
+    Used to detect if code is being run as a test or not
+    From: https://docs.pytest.org/en/5.1.1/example/simple.html#detect-if-running-from-within-a-pytest-run
+    """
+    import sys
```

Comment:
Why not shove this on `sunpy` instead?

Reply:
I didn't want a circular dependency given this is in the `__init__.py`. I didn't try it out here, but circular dependency is a possibility, I guess(?)

Reply:
This file has already imported `sunpy` and I don't think importing it in `sunpy.data.__init__` will cause problems. I think you should be ok.